.travis.yml

```diff
@@ -0,0 +1,24 @@
+services:
+- docker
+script:
+- docker build --target travistest --tag todo-app:travis .
+- docker run --mount type=bind,source="$(pwd)"/todo_app,target=/todo_app todo-app:travis
+  tests
+- docker run --env TRELLO_KEY --env TRELLO_TOKEN --env SECRET_KEY --env TRELLO_TODO_BOARDID
+  --mount type=bind,source="$(pwd)"/todo_app,target=/todo_app todo-app:travis tests_e2e
```

Comment:
Rather than mounting a directory, it's a bit simpler and neater to `COPY` the files into the test image.

Mounting is only when you want to keep the host filesystem and the container's in sync.